themes/bootstrap3/templates/Recommend/LibGuidesProfile.phtml

```diff
@@ -0,0 +1,20 @@
+<?php $account = $this->recommend->getResults(); ?>
+<?php if (!empty($account) && !empty($account->profile->url) && !empty($account->first_name)): ?>
+  <div class="libguides-profile">
+    <h4><?=$this->transEsc('libguides_profile_suggestion_heading')?></h4>
+    <a href="<?=$this->escapeHtmlAttr($account->profile->url)?>" target="_blank">
+      <?php if (!empty($account->profile->image)): ?>
+        <img class="libguides-profile-img" src="<?=$this->escapeHtmlAttr($account->profile->image->url)?>" />
+      <?php endif; ?>
+      <div class="libguides-profile-name"><?=$this->escapeHtml($account->first_name) . ' ' . $this->escapeHtml($account->last_name)?></div>
```

Comment:
I'd suggest making this `$account->last_name ?? ''` just in case that value is missing somehow.

Reply:
Yes good catch, fixed.